apps/prairielearn/src/pages/instructorCourseAdminSettings/instructorCourseAdminSettings.ejs

```diff
@@ -17,58 +17,147 @@
+                      class="form-control" 
+                      id="short_name" 
+                      name="short_name" 
+                      value="<%= course.short_name %>" 
+                      disabled
+                    >
+                    <small class="form-text text-muted">
+                        The short name of the course. Often this is the course rubric
+                        and number (e.g., "MATH 101" or "PHYS 440").
+                    </small>
+                  </div>
+                  <div class="form-group">
+                    <label for="title">Title</label>
+                    <input 
+                      type="text"
+                      class="form-control" 
+                      id="title" 
+                      name="title" 
+                      value="<%= course.title %>" 
+                      disabled
+                    >
+                    <small class="form-text text-muted">
+                        This is the official title of the course, as given in the course catalog.
+                    </small>
+                  </div>
+                  <div class="form-group">
+                    <label for="institution">Institution</label>
+                    <input 
+                      type="text"
+                      class="form-control" 
+                      id="institution" 
+                      name="institution" 
+                      value="<%= institution.short_name %> (<%= institution.long_name %>)" 
+                      disabled
+                    >
+                    <small class="form-text text-muted">
+                      This is your academic institution (e.g., "University of Illinois").
+                    </small>
+                  </div>
+                  <div class="form-group">
+                    <label for="display_timezone">Timezone</label>
+                    <input 
+                      type="text"
+                      class="form-control" 
+                      id="display_timezone" 
+                      name="display_timezone" 
+                      value="<%= course.display_timezone %>" 
+                      disabled
+                    >
+                    <small class="form-text text-muted">
+                      The allowable timezones are from the tz database. It's best to use
+                       a city-based timezone that has the same times as you.
+                    </small>
+                  </div>
+                  <div class="form-group">
+                    <label for="path">Path</label>
+                    <input 
+                      type="text"
+                      class="form-control" 
+                      id="path" 
+                      name="path" 
+                      value="<%= course.path %>" 
+                      disabled
+                    >
+                    <small class="form-text text-muted">
+                      The path where course files can be found. 
```

Comment:
I don't love this description for course files path. I can continue to work on a better description here.